app/models/container_service.rb

```diff
@@ -27,10 +27,23 @@ def container_groups_count
     number_of(:container_groups)
   end
 
+  include EventMixin
   include Metric::CiMixin
 
   PERF_ROLLUP_CHILDREN = :container_groups
 
+  def event_where_clause(assoc = :ems_events)
+    case assoc.to_sym
+    when :ems_events
+      # TODO: improve relationship using the id
+      ["container_namespace = ? AND container_group_name IN (?) AND #{events_table_name(assoc)}.ems_id = ?",
```

Comment:
a. so I understand the events we want to show for a service are all of the events relevant to a pods targeted by the service?

b. Is the final query the result of a join? I wonder why for some columns we have the table name and for others we do not.